src/actions/index.js

```diff
@@ -391,17 +400,15 @@ function updateOutput(
      *
      */
     const payload = {
-        output: {id: outputComponentId, property: outputProp},
+        output: outputIdAndProp,
     };
 
     if (event) {
         payload.event = event;
     }
 
     const {inputs, state} = dependenciesRequest.content.find(
-        dependency =>
-            dependency.output.id === outputComponentId &&
-            dependency.output.property === outputProp
+        dependency => dependency.output === outputIdAndProp
```

Comment:
Only check the output id since that is what we now pass in `_dash-dependencies`.